RevenueCatUI/Templates/V2/Components/Root/RootView.swift

```diff
@@ -47,6 +47,11 @@ struct RootView: View {
                     )
                 )
                 .fixedSize(horizontal: false, vertical: true)
+
+                // This spacer is to prevent the bottom most view from expanding into the
+                // safe space. iOS will take the bottom view and expand it to fill the
+                // the safe space and we don't want anything there.
+                Spacer()
```

Comment:
Hmm I'm wondering, should we remove the `edgesIgnoringSafeArea` completely so the view doesn't extend over the safe area? Would that allow us to remove this? (I'm not sure about this, I might be wrong/missing something).

Reply:
Its still need other this this happens 👇 🤷‍♂️ 
<img width="510" alt="Screenshot 2025-01-29 at 5 17 17 AM" src="https://github.com/user-attachments/assets/29b27e4e-9dc5-4a7a-b2c8-3f1737945bad" />